app/assets/javascripts/services/alerts_center_service.js

```diff
@@ -488,6 +488,15 @@ function alertsCenterService(API, $q, $timeout, $document, $uibModal, $http) {
     }
   }
 
+  var providerLink = function(providerType, providerId, newAlert) {
```

Comment:
You no longer need the `providerType` argument
```suggestion
  var providerLink = function(providerId, newAlert) {
```